client/src/pages/login.tsx

```diff
@@ -49,10 +48,11 @@ export const Login: React.FC = () => {
   return (
     <Container
       style={{
-        height: "100vh",
-        display: "flex",
-        justifyContent: "center",
-        alignItems: "center",
+        backgroundColor: '#F2F2F2', // Changed background color
```

Comment:
Try to update the entire screen as it does not include the entire screen at the moment.
<img width="2240" alt="Screenshot 2024-03-19 at 15 59 36" src="https://github.com/DevsRide/mern-car-rent-app/assets/11690402/e8f350a8-5968-4bc6-9edb-9dab8a441f11">